README.md

```diff
@@ -202,7 +221,7 @@ docker-compose up -d
 ## Notes:
 
  - A 'purge' script/service will run once a day, pruning old state from rooms to save disk space,
-   and restarting the ``synapse`` service to fetch an up-to-date whitelist of servers.
+   and restarting the ``synapse`` service to fetch an up-to-date whitelist of servers if necessary.
 
```

Comment:
"Our" purger only kicks user and fetches the whitelist. The rest is done by the synapse built-in purgers